web/admin/src/ContentDetailsEditor.js

```diff
@@ -61,6 +135,60 @@ export default class ContentDetailsEditor extends PureComponent {
     }
   }
 
+  isCSV = () => {
+    if (this.props.content.type === "textCSV" || this.props.content.type === "webCSV" || this.props.content.type === "videoCSV") {
```

Comment:
This function can simply be
```
return ['textCSV', 'webCSV', 'videoCSV'].includes(this.props.content.type)
```

Any time you find yourself writing something like
```
if (a || b && c) {
  return true
} else {
  return false
}
```

remember that `a || b && c` is itself a boolean-typed expression, so you can just say `return a || b && c`